content/authors/unboxed.yml

```diff
@@ -0,0 +1,8 @@
+# display name
+title: Unboxed Software
+# github username / organization
+github: Unboxed-Software
+# twitter / x username
+twitter: unboxedsoftware
+# website
+website: https://www.beunboxed.com/
```

Comment:
Unboxed hasn't worked on these early lessons for a while. Most of the content there right now is from Foundation. I figure we should leave credit to Unboxed, though, because they created the course and are good partners.  

```
content % git blame -w -M -C -C --line-porcelain intro-to-reading-data.md | grep "^author " | sort | uniq -c | sort -rn
 128 author Mike MacCana
  65 author James Pacheco
   3 author Thomas Simmer
   3 author Dhiraj Chauhan
🤠 /Users/mikemaccana/Code/solana/course/content
content % git blame -w -M -C -C --line-porcelain intro-to-writing-data.md | grep "^author " | sort | uniq -c | sort -rn
 132 author Mike MacCana
  88 author James Pacheco
   3 author Dhiraj Chauhan
   2 author user
   2 author Thomas Simmer
```